teensy_audio_octo/platformio.ini

```diff
@@ -0,0 +1,14 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[env:teensy35]
+platform = teensy
+board = teensy35
+framework = arduino
```

Comment:
do we need the [Octows2811](https://registry.platformio.org/libraries/paulstoffregen/OctoWS2811) as dependency? 
[lib_deps](https://docs.platformio.org/en/stable/projectconf/section_env_library.html#lib-deps)